### ISSUE TITLE
question about code

### ISSUE THREAD

**Issue by supersonicMaclaurin** [peylnog/ContinuousSR#3]
I noticed that there are some initial predefined values in the code, and I want to know how they are calculated or based on what principles they are set
`
        cho1 = torch.tensor([0, 0.41, 0.62, 0.98, 1.13, 1.29, 1.64, 1.85, 2.36]).cuda()
        cho2 = torch.tensor([-0.86, -0.36, -0.16, 0.19, 0.34, 0.49, 0.84, 1.04, 1.54]).cuda()
        cho3 = torch.tensor([0, 0.33, 0.53, 0.88, 1.03, 1.18, 1.53, 1.73, 2.23]).cuda()
        
        self.gau_dict = torch.tensor(list(product(cho1, cho2, cho3))).cuda()
        self.gau_dict = torch.cat((self.gau_dict, torch.zeros(1,3).cuda()), dim=0) # shape:[344,3]
`

**Comment by peylnog** [comment]
> I noticed that there are some initial predefined values in the code, and I want to know how they are calculated or based on what principles they are set
> 
> ```
>     cho1 = torch.tensor([0, 0.41, 0.62, 0.98, 1.13, 1.29, 1.64, 1.85, 2.36]).cuda()
>     cho2 = torch.tensor([-0.86, -0.36, -0.16, 0.19, 0.34, 0.49, 0.84, 1.04, 1.54]).cuda()
>     cho3 = torch.tensor([0, 0.33, 0.53, 0.88, 1.03, 1.18, 1.53, 1.73, 2.23]).cuda()
>     self.gau_dict = torch.tensor(list(product(cho1, cho2, cho3))).cuda()
>     self.gau_dict = torch.cat((self.gau_dict, torch.zeros(1,3).cuda()), dim=0) # shape:[344,3]
> ```

Thank you for your interest. In fact, we have provided a very detailed description of the predefined covariance in our paper, as it is the core finding of our work—“Deep Gaussian Prior.” We discovered that Gaussian covariance in deep space follows a certain distribution pattern. Based on this pattern, we predefined a set of covariances, which is exactly what you referred to.

**Comment by supersonicMaclaurin** [comment]
Yes, I noticed that the paper mentions' Deep Gaussian Prior '
> a) Approximately 99% of σ2 x, σ2 y, and ρσxσy fall within the ranges of 0 ∼ 2.4, 0 ∼ 2.2, and −0.9 ∼ 1.5, respectively. b) The distributions of the three covariances generally follow a Gaussian distribution.

but I would like to know how it is converted to these predefined values. I'm sorry, but I may not be familiar with the relevant mathematical theories

**Comment by peylnog** [comment]
> Yes, I noticed that the paper mentions' Deep Gaussian Prior '
> 
> > a) Approximately 99% of σ2 x, σ2 y, and ρσxσy fall within the ranges of 0 ∼ 2.4, 0 ∼ 2.2, and −0.9 ∼ 1.5, respectively. b) The distributions of the three covariances generally follow a Gaussian distribution.
> 
> but I would like to know how it is converted to these predefined values. I'm sorry, but I may not be familiar with the relevant mathematical theories

Sorry, I didn’t quite understand your question clearly. The details of Deep Gaussian Prior are well explained in our paper. In fact, there aren’t many complex mathematical theories involved. We derived the prior from the large-scale clear images using an optimization-based mapping method (GauImage).
